src/TaggedInput.jsx

```diff
@@ -144,21 +144,23 @@ var TaggedInput = React.createClass({
   _handleRemoveTag: function (index) {
     var self = this, s = self.state, p = self.props;
 
-    if (p.onBeforeRemoveTag(index)) {
-      var removedItems = s.tags.splice(index, 1);
+    if (!p.onBeforeRemoveTag(index)) {
```

Comment:
@thmain True, but it also required that all of code be wrapped (and indented) by the `if` statement.
Negating the condition allows this `if` statement to become a guard clause, which in turn lets us `return` at the top of the function.